Source/Fuse.Reactive.JavaScript/ArrayMirror.uno

```diff
@@ -5,20 +5,20 @@ namespace Fuse.Reactive
 {
 	class ArrayMirror: ListMirror
 	{
-		object[] _items;
+		protected List<object> _items;
 
-		internal ArrayMirror(ThreadWorker worker, Scripting.Array arr): base(arr)
+		internal ArrayMirror(IMirror mirror, Scripting.Array arr): base(arr)
 		{
-			_items = new object[arr.Length];
-			for (int i = 0; i < _items.Length; i++)
-				_items[i] = worker.Reflect(arr[i]);
+			_items = new List<object>();
```

Comment:
Provide the capacity in the ctor. It avoids some allocations while calling `Add`.